bundles/aliyun/src/main/java/org/apache/gravitino/oss/credential/OSSTokenProvider.java

```diff
@@ -150,13 +151,13 @@ private String createPolicy(Set<String> readLocations, Set<String> writeLocation
               URI uri = URI.create(location);
               allowGetObjectStatementBuilder.addResource(getOssUriWithArn(arnPrefix, uri));
               String bucketArn = arnPrefix + getBucketName(uri);
-              // ListBucket
+              // OSS use 'oss:ListObjects' to list objects in a bucket while s3 use 's3:ListBucket'
```

Comment:
Thank you. This comment is very useful for other developers to know why the action used is different.